src/components/navbar/logoStyle.module.scss

```diff
@@ -0,0 +1,7 @@
+.textContainer {
+  display: flex;
+  flex-direction: row;
+  padding: 0 1rem;
+  gap: 3px !important;
```

Comment:
We could eventually have problems with the code reviewers for using !important. Please, keep them to a minimum.

Reply:
Yes, you are right .. I will remove it 🚀